### PR TITLE
add the load fail message for BokehJS

### LINKED ISSUE #9684: If BokehJS isn't available, error message should be displayed
I'm trying to use Bokeh behind a proxy that doesn't allow outbound connections to https://cdn.pydata.org, which means that BokehJS can't be downloaded.  I can't do bokeh.resources = inline, as I have been unable to install bokehjs and its dependencies using npm due to the same lack of connectivity.

When I try to display a figure, nothing happens, but when I inspect the element, I can see the root cause is that BokehJS is not available (screenshot below).  I am getting pydata.org whitelisted, as we are not able to whitelist github and therefore it is challenging to get npm dependencies to install properly for local installation.  I think this should resolve the issue, but is it possible to have an error message at this point instead of just not rendering the script.

Alternative would be to have jupyter-bokeh include the installation of bokehjs locally, but I don't know the rationale behind having it loaded from the CDN, so assume it doesn't do that for a specific reason.

OS: Amazon Linux 
Browser: Firefox 71.0
Python: 3.6.8
Jupyter lab version: 1.2.4
Bokeh: 1.3.4
jupyter-bokeh: 1.1.1

Screen Shot 2019-12-20 at 12 45 29 pm

### PR DESCRIPTION
**Pull request by ayushiitr2901**
This Pull Request is regarding issue #9684, "Failed to load" message is visible now if BokehJS does not load for any possible reason. I have fixed my previous mistakes and updated the test_templates.py as well. I hope this works well.

- [ ] issues: fixes #9684 
- [ ] tests added / passed
checked on my local virtual env.

### CODE REVIEW

File: bokeh/core/_templates/autoload_nb_js.js [hunk 1]
```diff
@@ -160,8 +160,11 @@
       if (el != null) {
         el.textContent = "BokehJS " + root.Bokeh.version + " successfully loaded.";
       }
-    } else if (Date.now() < root._bokeh_timeout) {
+    }
+    else if (Date.now() < root._bokeh_timeout) {
```
Comment (bryevdv) [inline bokeh/core/_templates/autoload_nb_js.js]: the `else if` should be moved back up one line right after the `}` to match our repo style

**Comment by bryevdv** [comment]
@ayushiitr2901 This is much improved. I left one style comment, and also there is one hard linter failure:
```

84 E       AssertionError: Code quality issues:
85 E         File contains tabs instead of spaces: bokeh/core/_templates/autoload_nb_js.js, line 167.
86 E       assert 1 == 0
87 E        +  where 1 = len(['File contains tabs instead of spaces: bokeh/core/_templates/autoload_nb_js.js, line 167.'])
```

We always uses spaces for indentation everywhere in every file in the repo, and enforce this via code checks. You just need to convert the tab on line 167 to be spaces instead.

File: bokeh/core/_templates/autoload_nb_js.js (hunk 1 above)
Comment (ayushiitr2901) [reply bokeh/core/_templates/autoload_nb_js.js]: I will fix it.

**Comment by bryevdv** [comment]
@ayushiitr2901 This looks great. Since this is not easily testable, by way of documentation, can you put a screenshot of the code path being employed in a comment here? (i.e. turning off your network access and trying to run a notebook should trigger it)

**Comment by bryevdv** [comment]
@ayushiitr2901 So I tried this out locally and I am afraid I was a little overly optimistic about what it would take to fix this. I have a better idea now and will write up some comments soon (we are in the middle of a release so it may be a few days). Otherwise in the mean time you might look at the `autoload_js.js` template. I think the `on_error` function will need to set a global flag if there is an error, an then *after* the `load_libs` call, that's when it  can be checked.

**Comment by bryevdv** [comment]
Absent further feedback I am going to go ahead and close this PR

**Pull request closed without merging**